### PR TITLE
grafana-dash-n-grab: init at 0.3.1

### PR DESCRIPTION
**Pull request by wraithm**
###### Description of changes

New package grafana-dash-n-grab (bin: gdg) 0.3.1

Homepage: https://github.com/esnet/gdg

Grafana Dash-n-Grab (GDG) -- Dashboard/DataSource Manager. The purpose of this project is to provide an easy to use CLI to interact with the grafana API allowing you to backup and restore dashboard, datasources, and other entities.

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/monitoring/grafana-dash-n-grab/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  vendorSha256 = "sha256-pXo80Tean5OkQ0Sv2+/RlRnAtaClwGP7tuDC2irsh+E=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.Version=${version}"
+    "-X github.com/esnet/gdg/version.GitCommit=${src.rev}"
+  ];
+
+  doCheck = false;
```
Comment (wraithm) [inline pkgs/servers/monitoring/grafana-dash-n-grab/default.nix:23]: The test suite tries to communicate with a running version of grafana locally... It fails if you don't have grafana running.

**Comment by wraithm** [comment]
@ofborg build grafana-dash-n-grab

File: pkgs/servers/monitoring/grafana-dash-n-grab/default.nix (hunk 1 above)
Comment (cdepillabout) [reply pkgs/servers/monitoring/grafana-dash-n-grab/default.nix:23]: Could you add this as a comment above this line?

**Comment by cdepillabout** [comment]
cc @Ma27 and @Frostman for a potential review on this.  (I think you guys have reviewed grafana-related PRs in the past.)

**Comment by cdepillabout** [comment]
I think this PR looks good.  It builds and appears to work.

I'm going to go ahead and merge it in, but if anyone ends up taking a look at this and sees something to improve, we can fix it up in a follow-up PR.

**Pull request merged**